recipes/subversion/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "subversion" %}
+{% set version = "1.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.apache.org/subversion/subversion-{{ version }}.tar.bz2
+  sha256: bc50ce2c3faa7b1ae9103c432017df98dfd989c4239f9f8270bb3a314ed9e5bd
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf  # [unix]
+    - libtool  # [unix]
+    - pkg-config  # [unix]
+    - nasm
+    - make
+    - python >=2.7,<3
+  host:
+    - boost-cpp
+    - libapr
+    - libaprutil
+    - sqlite
+    - zlib
+    - openssl
+    - libdb
+    - lz4-c
+    - libutf8proc
+    - gettext
+    - expat
+    - gnupg  # [unix]
+    - cyrus-sasl  # [unix]
+    - libmagic  # [unix]
+    - serf  # [linux]
+  run:
+    - libapr
```

Comment:
Are these libraries missing a run_exports?

Reply:
Probably - that would explain why I need to put them in run deps

Reply:
Both [`libapr`]( https://github.com/conda-forge/apr-feedstock/blob/f2b2554770474d280b0c67d26eefc2f75f6f04d2/recipe/meta.yaml#L44-L47 ) and [`libaprutil `]( https://github.com/conda-forge/apr-feedstock/blob/f2b2554770474d280b0c67d26eefc2f75f6f04d2/recipe/meta.yaml#L71-L72 ) have `run_exports`.